geometry/geometry.py

```diff
@@ -134,25 +134,27 @@ def max_parts(self, num_cuts: float) -> float:
 
         >>> circle = Circle(5)
         >>> circle.max_parts(0)
-        1.0
+        1
```

Comment:
why do you wanna add a ".0" is the rest of test cases ???

Reply:
I have removed the `.0` at the end because the result is supposed to be a whole number, not a float.